Marlin/Regulator.h

```diff
@@ -33,9 +40,25 @@
 #endif // E_REGULATOR_SENSOR
 #define REGULATOR_LOW_P 2
 
+#define REGULATOR_PROTECTION_TIME_S (5)
+#define REGULATOR_PROTECTION_BAND   (4)
+
+#define REGULATOR_NOT_PRESENT_VALUE (130.5)
```

Comment:
@danthompson41 have you tried this on a few printers? I feel like this isn't consistent, but I could be wrong. Is this just the max-value that the ADC reports?